go.sum

```diff
@@ -742,6 +740,14 @@ github.com/spf13/viper v1.3.2/go.mod h1:ZiWeW+zYFKm7srdB9IoDzzZXaJaI5eL9QjNiN/DM
 github.com/spf13/viper v1.7.0/go.mod h1:8WkrPz2fc9jxqZNCJI/76HCieCp4Q8HaLFoCha5qpdg=
 github.com/stretchr/objx v0.1.0/go.mod h1:HFkY916IF+rwdDfMAkV7OtwuqBVzrE8GR6GFx+wExME=
 github.com/stretchr/objx v0.1.1/go.mod h1:HFkY916IF+rwdDfMAkV7OtwuqBVzrE8GR6GFx+wExME=
+github.com/stretchr/testify v1.2.2/go.mod h1:a8OnRcib4nhh0OaRAV+Yts87kKdq0PP7pXfy6kDkUVs=
```

Comment:
There are so many version of testify, Have you tried  to run command `go mod tidy`?

Reply:
Yes, I have run ```go mod tidy``` and our CI also will check it.

Reply:
I use the ```go mod graph``` and I find the old version is from the other third library.

```
> $ go mod graph  |grep testify                                                                                                                                                                                
github.com/pingcap/tidb github.com/stretchr/testify@v1.7.2-0.20220504104629-106ec21d14df
github.com/Azure/azure-sdk-for-go/sdk/azcore@v0.20.0 github.com/stretchr/testify@v1.7.0
github.com/Azure/azure-sdk-for-go/sdk/internal@v0.8.1 github.com/stretchr/testify@v1.7.0
github.com/Azure/azure-sdk-for-go/sdk/storage/azblob@v0.2.0 github.com/stretchr/testify@v1.7.0
github.com/HdrHistogram/hdrhistogram-go@v1.1.2 github.com/stretchr/testify@v1.7.0
github.com/Shopify/sarama@v1.29.0 github.com/stretchr/testify@v1.7.0
github.com/cockroachdb/pebble@v0.0.0-20210719141320-8c3bd06debb5 github.com/stretchr/testify@v1.6.1
github.com/dgraph-io/ristretto@v0.1.1-0.20220403145359-8e850b710d6d github.com/stretchr/testify@v1.4.0

...
```